atlasdb-ete-test-utils/src/main/java/com/palantir/atlasdb/ete/Gradle.java

```diff
@@ -42,10 +42,10 @@ public static Gradle ensureTaskHasRun(String command) {
     @Override
     protected void before() throws Throwable {
         if (isRunningOutsideOfGradle()) {
-            System.out.println("It looks like you are not running in gradle, performing the required gradle command: " + command);
+            System.out.println("It looks like you are not running in gradle, performing the required gradle command: " + command);  // (authorized)
```

Comment:
I was vague there.  I was specifically asking about the purpose of the "// (authorized)".